continuousscanning/gvks.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python3
```

Comment:
What is this file?

Reply:
It generates a list of GVKs that we should monitor. I debated whether to include it in the Continuous Scanning tree or not.

Reply:
You could have it in this tree, but maybe have it nested under a "test" directory or something so it will be clear why it is here.